<launch>

	<node pkg="main_loop" type="add_two_ints_client.py" name="add_two_ints_client" output="screen" respawn="true">
	</node>
	<node pkg="main_loop" type="add_two_ints_server.py" name="add_two_ints_server" output="screen" respawn="true" >
	</node>






</launch>
